roles/cleanup_gs_data/templates/cleanup_gs.bash

```diff
@@ -0,0 +1,39 @@
+# If folder is empty for 1 week -> mail helpdesk
+# If folder is empty for 2 weeks -> delete
+# If folder is not empty, but no .finished file is present (1 week) -> mail helpdesk
+# If folder is not empty, but no .finished file is present (2 weeks) -> delete
+
+#TODO: notifications and removal of folders
+
+#!/bin/bash
+
+dirToCheck="/groups/umcg-genomescan/"*
+dateInSecNow=$(date +%s)
+
+# Check only dirs, ignore files
+for dir in $(find ${dirToCheck} -maxdepth 1 -type d)
```

Comment:
```suggestion
for dir in $(find ${dirToCheck}  -mindepth 1 -maxdepth 1 -type d)
```

Reply:
You also need `-mindepth 1` to prevent getting the ${dirToCheck} in the results as well.